diff --git a/target/linux/armvirt/image/Makefile b/target/linux/armvirt/image/Makefile
index 73d9145e..c5401cce 100644
--- a/target/linux/armvirt/image/Makefile
+++ b/target/linux/armvirt/image/Makefile
@@ -25,4 +25,26 @@ define Image/Build
 		gzip -9n >$(BIN_DIR)/$(IMG_PREFIX)-root.$(1).gz
 endef
 
+define Device/Phicomm-n1
+  DEVICE_MODEL := Phicomm-n1
+  DEVICE_PACKAGES := \
+	cypress-firmware-43430-sdio \
+	cypress-nvram-43430-sdio-rpi-3b \
+	cypress-firmware-43455-sdio \
+	cypress-nvram-43455-sdio-rpi-3b-plus \
+	kmod-brcmfmac wpad-basic \
+	bash vim install-program lm-sensors lm-sensors-detect \
+	htop lscpu iperf3 fdisk lsblk parted blkid losetup \
+	kmod-fs-ext4 kmod-fs-vfat kmod-fs-exfat ntfs-3g \
+	e2fsprogs dosfstools ntfsprogs_ntfs-3g \
+	kmod-usb-storage kmod-usb-storage-extras kmod-usb-storage-uas \
+	kmod-usb-net kmod-usb-net-asix kmod-usb-net-asix-ax88179 \
+	kmod-usb-net-rtl8150 kmod-usb-net-rtl8152 \
+	luci-app-samba4 luci-app-cifs-mount luci-app-docker \
+	luci-theme-bootstrap-mod
+endef
+ifeq ($(SUBTARGET),64)
+	TARGET_DEVICES += Phicomm-n1
+endif
+
 $(eval $(call BuildImage))
